src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandler.java

```diff
@@ -222,7 +223,18 @@ private SscsCaseData updateCaseAfterReinstatementGranted(SscsCaseData caseData,
         return caseData;
     }
 
+    private boolean hasFtaBeenChosenAsOneOfThePartyMembers(SscsCaseData caseData) {
+        String confidentialityType = caseData.getConfidentialityType();
+        List<String> confidentialityPartyMembers = caseData.getConfidentialityPartyMembers();
 
+        if (isNull(confidentialityType)
+            || isNull(confidentialityPartyMembers)
+            || "general".equalsIgnoreCase(confidentialityType)) {
```

Comment:
Avoid literal, declare as static var at top of class (or pull in from elsewhere).